tests/flow/base.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+import redis
+
+
+class FlowTestsBase(unittest.TestCase):
+    RUN_TIME_ROW_INDEX = 4
+
+    @classmethod
+    def setUpClass(cls):
+        cls.redis_con = redis.Redis(host='localhost', port=6379)
+
+    def _assert_only_expected_resuls_are_in_actual_results(self,
+                                                           actual_result,
+                                                           query_info):
+        # assert num resuls
+        num_results = len(actual_result.result_set[1:])
+        self.assertEqual(num_results, len(query_info.expected_result))
+
+        # assert actual values vs expected values
+        for res in query_info.expected_result:
+            self.assertTrue(res in actual_result.result_set[1:],
+                            'The item %s is NOT in the actual result' + str(res))
+
+    def _assert_actual_results_contained_in_expected_results(self,
+                                                             actual_result,
+                                                             query_info,
+                                                             num_contained_results):
+        # assert num results
+        num_results = len(actual_result.result_set[1:])
+        self.assertEqual(num_results, num_contained_results)
+
+        # assert actual values vs expected values
+        expected_result = query_info.expected_result
+        count = len([res for res in expected_result if res in actual_result.result_set[1:]])
+
+        # assert number of different results is as expected
+        self.assertEqual(count,
+                         num_contained_results,
+                         'The actual result is: ' + str(actual_result.result_set[1:]))
+
+    def _assert_run_time(self, actual_result, query_info):
+        if query_info.max_run_time_ms is not None:
+            run_time_string = actual_result.statistics[self.RUN_TIME_ROW_INDEX]
+            actual_run_time = float(run_time_string.split(': ')[1].split(' milliseconds')[0])
```

Comment:
It would be nice if redisgraph-py would extracted the query runtime to its own field of type float.